http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AdminServiceInterp.scala

```diff
@@ -42,10 +41,10 @@ final class AdminServiceInterp[F[_]: Parallel](authProvider: LeoAuthProvider[F],
       _ <- F.raiseWhen(!hasPermission)(NotAnAdminError(userInfo.userEmail, Option(ctx.traceId)))
 
       // Find the config relevant to this cloud and app type
-      appConfig: KubernetesAppConfig <- adminAppConfig.configForTypeAndCloud(req.appType, req.cloudProvider) match {
-        case Some(conf) => F.pure(conf)
-        case None       => F.raiseError(NoMatchingAppError(req.appType, req.cloudProvider, Option(ctx.traceId)))
-      }
+      appConfig: KubernetesAppConfig <- F.fromOption(
```

Comment:
Ahhh scala, every time I get comfortable with something (e.g. match) I then realize that there is almost always something a lot better to use instead 😓

Reply:
Heh - either is fine honestly, I happen to like `fromOption` :)